demos/common/demo_runner/aws_demo_runner.c

```diff
@@ -57,7 +61,14 @@
  */
 void DEMO_RUNNER_RunDemos( void )
 {
-    /* vStartMQTTDemo( ); */
+    /* These demos are shared with the C SDK and perform their own initialization and cleanup. */
+
+    /* vStartMQTTDemo(); */
+
+    /* These demos require the libraries to be initialized before they run. */
+    configASSERT( IotMqtt_Init() == IOT_MQTT_SUCCESS );
```

Comment:
Does the MQTT demo not require these libs to be initialized?

Reply:
The new demos perform the initialization themselves, as they're shared with the C SDK.